cuda_core/cuda/core/experimental/_graph.py

```diff
@@ -687,7 +687,7 @@ def add_child(self, child_graph: GraphBuilder):
             [
                 handle_return(
                     driver.cuGraphAddChildGraphNode(
-                        graph_out, deps_info_out[0], num_dependencies_out, child_graph._mnff.graph
+                        graph_out, *deps_info_out, num_dependencies_out, child_graph._mnff.graph
```

Comment:
It was done like this because during CUDA 13 bringup @rwgk wanted to make cuda-core work for both 12 and 13 (which have different signatures) without revealing anything about 13 (https://github.com/NVIDIA/cuda-python/pull/722). @vzhurba01 [expressed concerns](https://github.com/NVIDIA/cuda-python/pull/722#issuecomment-3024923443) that make sense to me. Now that we are bitten by this and that 13 is out, we should properly check the binding version and not try to hide the differences.